src/debugger/src/cli_debugger.cpp

```diff
@@ -0,0 +1,154 @@
+#include "debugger/cli_debugger.h"
+#include <iostream>
+
+using namespace debugger;
+using namespace std;
+
+CliDebugger::CliDebugger(std::unique_ptr<DebuggerCore> debugger_core)
+    : debugger_core(move(debugger_core)),
+      command_parser("tdb", "The tvp Debugger") {
+
+	// clang-format off
+	command_parser.add_options()
+	    ("run","Run the ROM till it encounters a breakpoint")
+	    ("step", "Execute one System tick")
+	    ("bp-set", "Set a breakpoint",cxxopts::value<string>()->default_value("NULL"))
```

Comment:
An empty string is perhaps a better choice for a default value, it's best to avoid `NULL`

Reply:
It lets you use `.empty()` as well